_includes/2020/templates/Head.php

```diff
@@ -13,6 +13,9 @@ static function render($fields = []) {
       if(!isset($fields->title)) {
         $fields->title = 'Keyman | Type to the world in your language';
       }
+      if(!empty($fields->language)) {
```

Comment:
oops - fixing last suggestion
```suggestion
      if(empty($fields->language()) {
```